array-api-skips.txt

```diff
@@ -0,0 +1,133 @@
+# copy not implemented
+array_api_tests/test_creation_functions.py::test_asarray_arrays
+
+# https://github.com/numpy/numpy/issues/20870
+array_api_tests/test_data_type_functions.py::test_can_cast
+
+# The return dtype for trace is not consistent in the spec
+# https://github.com/data-apis/array-api/issues/202#issuecomment-952529197
+array_api_tests/test_linalg.py::test_trace
+
+# waiting on NumPy to allow/revert distinct NaNs for np.unique
+# https://github.com/numpy/numpy/issues/20326#issuecomment-1012380448
+array_api_tests/test_set_functions.py
+
+# newaxis not included in numpy namespace as of v1.26.2
+array_api_tests/test_constants.py::test_newaxis
+
+# linalg.solve issue in numpy.array_api as of v1.26.2 (see numpy#25146)
```

Comment:
This issue has been fixed already, and didn't affect the main namespace. Can this be dropped?

Reply:
Yes, removed.